internal/installation/appinfo.go

```diff
@@ -44,6 +48,14 @@ func newExecFromDir(baseDir string, exec executableType) (string, error) {
 		path = filepath.Dir(osutils.Executable())
 	}
 
+	// Work around dlv debugger giving an unexpected executable path
+	if !condition.BuiltViaCI() && len(os.Args) > 1 && strings.Contains(os.Args[0], "__debug_bin") {
+		rootPath := filepath.Clean(environment.GetRootPathUnsafe())
+		if rootPath == filepath.Clean(path) {
+			path = filepath.Join(path, "build")
+		}
+	}
+
```

Comment:
This feels like a sort of pollution. Is it OK?

Reply:
It's a bit awkward but edge case and safe enough IMO.